cms/djangoapps/contentstore/views/tests/utils.py

```diff
@@ -59,25 +59,37 @@ def validate_preview_html(self, xblock, view_name, can_add=True, can_reorder=Tru
             '<span data-tooltip="Drag to reorder" class="drag-handle action"></span>',
             can_reorder
         )
+
+        move_action_html = '<a class="move-button" href="#" role="button">Move</a>'
+        delete_action_html = '<a class="delete-button" href="#" role="button">Delete</a>'
+        duplicate_action_html = '<a class="duplicate-button" href="#" role="button">Duplicate</a>'
+        if in_unit:
+            move_action_html = '<button data-tooltip="Move" class="btn-default move-button action-button">'
+            delete_action_html = '<button data-tooltip="Delete" class="btn-default delete-button action-button">'
+            duplicate_action_html = \
+                '<button data-tooltip="Duplicate" class="btn-default duplicate-button action-button">'
```

Comment:
Nit: I think this would be more clear as:

```python
    if in_unit:
        move_action_html = '<button data-tooltip="Move" class="btn-default move-button action-button">'
        delete_action_html = '<button data-tooltip="Delete" class="btn-default delete-button action-button">'
        duplicate_action_html = \
            '<button data-tooltip="Duplicate" class="btn-default duplicate-button action-button">'
    else:
        move_action_html = '<a class="move-button" href="#" role="button">Move</a>'
        delete_action_html = '<a class="delete-button" href="#" role="button">Delete</a>'
        duplicate_action_html = '<a class="duplicate-button" href="#" role="button">Duplicate</a>'
```

Reply:
Updated it.